source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md

```diff
@@ -0,0 +1,151 @@
+| glueJobProps?       | [`cfnJob.CfnJobProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnJobProps.html)                    | User provided props to override the default props for the CfnJob.                                                                                                                                                                                                                     |
+| existingGlueJob?    | [`cfnJob.CfnJob`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnJob.html)                              | Existing CfnJob configuration, if this this set then glueJobProps is ignored.                                                                                                                                                                                                         |
+| fieldSchema?        | [`CfnTable.ColumnProperty[]`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnTable.ColumnProperty.html) | Glue Database for this construct. If not provided the construct will create a new Glue Database. The database is where the schema for the data in Kinesis Data Streams is stored                                                                                                      |
+| database?           | [`CfnDatabase`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnDatabase.html)                           | Glue Database for this construct. If not provided the construct will create a new Glue Database. The database is where the schema for the data in Kinesis Data Streams is stored                                                                                                      |
+| table?              | [`CfnTable`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnTable.html)                                 | Glue Table for this construct, If not provided the construct will create a new Table in the database. This table should define the schema for the records in the Kinesis Data Streams. Either @table or @fieldSchema is mandatory. If @table is provided then @fieldSchema is ignored |
+| outputDataStore?    | [`SinkDataStoreProps`](#sinkdatastoreprops)                                                                                   | The output datastore properties where the ETL output should be. Current datastore types suported is only S3. written                                                                                                                                                                  |
```

Comment:
Incomplete sentence or typo at the end `written`

Reply:
Yes, good catch, will be removed.